srcCpp03/RTIDDSImpl.cxx

```diff
@@ -857,9 +857,9 @@ class RTIPublisherBase: public IMessagingWriter {
         return (unsigned int)_writer->datawriter_protocol_status().pulled_sample_count().total();
     }
 
-    void wait_for_acknowledgments(const dds::core::Duration & timeout){
+    void wait_for_acknowledgments(long sec, unsigned long nsec){
```

Comment:
Could you add a space between ")" and "{"

Reply:
Done